packages/atomic/src/components/search/atomic-search-box/e2e/page-object.ts

```diff
@@ -6,6 +6,10 @@ export class SearchBoxPageObject extends BasePageObject<'atomic-search-box'> {
     super(page, 'atomic-search-box');
   }
 
+  get component() {
+    return this.page.locator('atomic-search-box');
```

Comment:
Here, I use a locator because I want to capture a very specific element that in itself, to set an attribute on it ;)